blocitoff/app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
@jboltz88 `UsersHelper` is another skeleton auto-generated file that can be safely removed.